recipes/configs/llama2/generation_v2.yaml

```diff
@@ -9,6 +9,10 @@
 # Model arguments
 model:
   _component_: torchtune.models.llama2.llama2_7b
+# You can turn uncomment the following lines to enable quantization for faster inference and potentially lower VRAM
+# quantization_method:
+#   _component_: torchao.quantization.quant_api.int4_weight_only # int4_weight_only is a good balance of speed and memory
+#   use_hqq: False # Turn on to use Half-Quadratic Quantization
```

Comment:
what does it mean? Can you add if it makes it faster/more accurate/less memory?

Reply:
sorry, what i meant is that this should be made clear for the user in the comment :P